xonsh/xontribs.json

```diff
@@ -43,6 +43,14 @@
   "url": "https://github.com/AstraLuma/xontrib-avox",
   "description": ["Automatic (de)activation of virtual environments as you cd around"]
  },
+ {"name": "back2dir",
+  "package": "xontrib-back2dir",
+  "url": "https://github.com/anki-code/xontrib-back2dir",
+  "description": [
+    "Return to the most recently used directory when starting xonsh shell. ",
```

Comment:
"starting xonsh" or "starting the xonsh shell"

Reply:
thanks!